PurchasesTests/PurchasesTests-Bridging-Header.h

```diff
@@ -23,4 +23,5 @@
 #include <Purchases/RCEntitlementInfo.h>
 #include <Purchases/RCEntitlementInfos.h>
 #include <Purchases/RCDeviceCache.h>
+#include <Purchases/RCDeviceCache+Protected.h>
```

Comment:
for testing purposes. 
the good news here is that the class can still be private, since you can use `@testable import Purchases` to access private classes.